### ISSUE TITLE
Some of app elements (e.g Profile menu items, trans history list items etc) periodically stop responding on Android app

### ISSUE THREAD

**Issue by pavloburykh** [status-im/status-react#14020]
Currently there are no specific steps to reproduce the bug it occurs sporadically. I have faced it several times already on Android builds in these PRs [1](https://github.com/status-im/status-mobile/pull/13858) and [2](https://github.com/status-im/status-mobile/pull/13883). Will proceed  looking at it on nightly and give updates here. 

The last time I faced the bug my **steps were the following:**  (these steps are unlikely to work out again)

1. Restored the account that has balance and some history of transactions
2. Right after restore went to Wallet tab - opened history tab - waited until history was loaded
3. Opened details of couple of transactions
4. Went back to account page
5. Opened send transaction screen, opened receive transaction screen
6. Backgrounded app
7. Reopened app
8. Went to profile tab
9. Opened Add contact screen
10. Closed add contact screen
11. Tried to enter some other Profile pages

**Actual result:** Profile page menu items stopped responding as well as many others. 

Repeating the above steps does not guarantee reproduction. I Attach log file below.  

https://user-images.githubusercontent.com/97245802/190633966-e3cdd9b8-e6aa-43fb-9bfb-2dcdb2c9614b.mp4

 
[geth.log](https://github.com/status-im/status-mobile/files/9583761/geth.log)

**OS:** Android 11
**Device:** Samsung Galaxy A52

![photo_2022-09-16 14 59 44](https://user-images.githubusercontent.com/97245802/190634139-60dfaf8b-d2e6-4b54-8376-d310be36ac75.jpeg)

**Comment by flexsurfer** [comment]
most likely its RN bridge, react logs would be useful

**Comment by pavloburykh** [comment]
> most likely its RN bridge, react logs would be useful

thanx @flexsurfer I've left geth.log in description above. Duplicate here [geth.log](https://github.com/status-im/status-mobile/files/9583761/geth.log)

**Comment by flexsurfer** [comment]
yeah but its a status-go log, but we need status-react log

**Comment by pavloburykh** [comment]
> yeah but its a status-go log, but we need status-react log

oh, sorry for that @flexsurfer , I've misunderstood. Here is RN logfile 
[Status.log](https://github.com/status-im/status-mobile/files/9585320/Status.log)

**Comment by Parveshdhull** [comment]
This looks somewhat related to navigation. I remember seeing somewhat similar behavior when having a different root-id or something.
Some errors like `can't set stack with id 0` etc in metro. and navigation didn't work.

**Comment by Parveshdhull** [comment]
(or something like `unhandled exception, can't set ... of unmounted component`)

**Comment by qoqobolo** [comment]
There is a way to reproduce the bug more likely: you just need to tap menu settings in the profile and the `profile` tab itself. If you do it fast enough, it will lead to a crash.

Crash:
1. Open Profile
2. Start tapping menu items and the profile tab icon quickly until the app gets frozen 
3. Put the app in the background and re-open it

https://user-images.githubusercontent.com/67952253/191244851-83011043-5e1b-4ec4-bb74-2c5b0119fbcb.mp4

The case when the app stops responding:

https://user-images.githubusercontent.com/67952253/191245347-8e22fbc5-b174-47ac-a371-4ec084a72113.mp4

[geth.log](https://github.com/status-im/status-mobile/files/9607057/geth.log)
[Status.log](https://github.com/status-im/status-mobile/files/9607058/Status.log)

**Comment by Parveshdhull** [comment]
Hi @qoqobolo, Thank you very much for the reproduction video.
Is this issue only exist in old ui, or also in new ui?

**Comment by qoqobolo** [comment]
Hey @Parveshdhull , I'm not sure I can reproduce it with the same steps in new UI, as currently tabs stop displaying when you open some of the settings. With a quick press of the back button I was unable to reproduce the issue.

https://user-images.githubusercontent.com/67952253/193565532-3bc59116-9acb-483a-a475-56d60b623af8.mp4

**Comment by Parveshdhull** [comment]
Thank you for confirming.

**Comment by pavloburykh** [comment]
Issue is still reproducible as for nightly 03 Jan 2023. 
Will be looking for steps and provide an update

https://user-images.githubusercontent.com/97245802/210413115-517558ea-816f-4bae-b634-0d2d9d5b6cce.mp4

**Comment by flexsurfer** [comment]
hey @Parveshdhull i'll take this one, any thoughts or comments?

**Comment by Parveshdhull** [comment]
Hi, I have seen this kind of behavior due to the RNN's unhandled promise. Probably trying to dismiss models that don't exist and are similar while popping to root.

Maybe this issue doesn't exist anymore, probably fixed in https://github.com/status-im/status-mobile/pull/14811. (@status-im/mobile-qa, please can you check)

Also, I think we should remove the old navigation code. We have events like `change-tab` at several places for tabs that don't even exist. That might create this kind of behavior.

**Comment by pavloburykh** [comment]
> Maybe this issue doesn't exist anymore, probably fixed in #14811. (@status-im/mobile-qa, please can you check)

@flexsurfer @Parveshdhull as there are no specific steps of reproduction the bug, let's wait for some time after #14811 will be tested and merged into develop. If we will not be facing the issue again - I will close this ticket. WDYT?

**Comment by qoqobolo** [comment]
Closing the issue since it's not reproducible atm in develop nor in https://github.com/status-im/status-mobile/pull/14811.
Please, reopen if you encounter the problem again at some point.

**Comment by pavloburykh** [comment]
Still reproducing the issue. Last reproduction on nightly from 27.01.2023
Re-opening the issue.


https://user-images.githubusercontent.com/97245802/215139958-c5e48693-77ee-4afa-82bc-20e88a1cf50f.mp4

[Status-debug-logs (24).zip](https://github.com/status-im/status-mobile/files/10520947/Status-debug-logs.24.zip)
